### ISSUE TITLE
IDisposable Reponse is not disposed

### ISSUE THREAD

**Issue by blazepasquale** [CoreWCF/CoreWCF#932]
Hi,

in WCF responses which implement the IDisposable interfaces are automatically disposed. In CoreWCF that does not work. I have to manually implement in each operation following workaround:

```
OperationContext.Current.OperationCompleted += delegate
{
    response?.Dispose();
};
```

Is this a known bug? Could i move this workaround to a general behavior?

Thanks!

**Comment by mconnew** [comment]
Is the response here an object you are returning from your service method? That should be working so if it isn't, that's a bug. Make sure that a second call to Dispose doesn't throw an exception and just no-ops so that when this is fixed a double Dispose doesn't cause problems.

**Comment by blazepasquale** [comment]
Yes it is an object. I understood that the OperationBehaviorAttribute is automatically set to each operation. There is a property "AutoDisposeParameters". This is by default false - in wcf it was by default true. If I set this to true my return object is automatically disposed.

**Comment by zabulus** [comment]
There is definetly a bug with auto dispose. We have following service
```
public interface IServer
{
    [CoreWCF.OperationContract]
    Task<Stream> ExecuteGenerationAsync(object data);
}
```
And we have a static RecyclableMemoryStreamManager that is used to reduce new allocations of MemoryStream objects. Stream returned from this method is not disposed and hence not returned to RMSM.
